core/cache/suite_test.go

```diff
@@ -0,0 +1,12 @@
+package cache_test
+
+import (
+	"os"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
what's this for?

Reply:
Left there, initially i did some bootstrapping here.